.github/CODEOWNERS

```
@@ -1 +1,2 @@
 site/ @coder/frontend
+docs/ @ammario
```

```suggestion
docs/ @ammario @jsjoeio 
```

feel free to add me as well